playwright.config.ts

```diff
@@ -22,7 +22,7 @@ export default defineConfig<PluginOptions>({
   /* Retry on CI only */
```

Comment:
These changes seems to require to be reversed (opt out, deleted space, etc)

Reply:
Fixed